--- old/config.def.h	2019-03-06 11:44:08.828086459 +0100
+++ new/config.def.h	2019-03-06 11:44:32.016087010 +0100
@@ -5,13 +5,13 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-static char *font = "Liberation Mono:pixelsize=12:antialias=true:autohint=true";
-static int borderpx = 2;
+static char *font = "kiwi:size=7";
+static int borderpx = 20;
 
 /* disable bold, italic and roman fonts */
-int disablebold = 0;
-int disableitalic = 0;
-int disableroman = 0;
+int disablebold = 1;
+int disableitalic = 1;
+int disableroman = 1;
 
 /*
  * What program is execed by st depends of these precedence rules:
@@ -21,7 +21,7 @@
  * 4: value of shell in /etc/passwd
  * 5: value of shell in config.h
  */
-static char *shell = "/bin/sh";
+static char *shell = "/bin/bash";
 char *utmp = NULL;
 char *stty_args = "stty raw pass8 nl -echo -iexten -cstopb 38400";
 
@@ -54,7 +54,7 @@
  * blinking timeout (set to 0 to disable blinking) for the terminal blinking
  * attribute.
  */
-static unsigned int blinktimeout = 800;
+static unsigned int blinktimeout = 0;
 
 /*
  * thickness of underline and bar cursors
@@ -90,41 +90,40 @@
 /* Terminal colors (16 first used in escape sequence) */
 static const char *colorname[] = {
 	/* 8 normal colors */
-	"black",
-	"red3",
-	"green3",
-	"yellow3",
-	"blue2",
-	"magenta3",
-	"cyan3",
-	"gray90",
+	"#021B21",
+	"#C2454E",
+	"#7CBF9E",
+	"#8A7A63",
+	"#2E3340",
+	"#FF5879",
+	"#44B5B1",
+	"#F2F1B9",
 
 	/* 8 bright colors */
-	"gray50",
-	"red",
-	"green",
-	"yellow",
-	"#5c5cff",
-	"magenta",
-	"cyan",
-	"white",
+	"#234149",
+	"#EF5847",
+	"#A2D9B1",
+	"#BEB090",
+	"#61778D",
+	"#FF99A1",
+	"#9ED9D8",
+	"#F6F6C9",
 
 	[255] = 0,
 
 	/* more colors can be added after 255 to use with DefaultXX */
-	"#cccccc",
-	"#555555",
+	"#E8DFD6",
+	"#021B21"
 };
 
-
 /*
  * Default colors (colorname index)
  * foreground, background, cursor, reverse cursor
  */
-unsigned int defaultfg = 7;
-unsigned int defaultbg = 0;
+unsigned int defaultfg = 256;
+unsigned int defaultbg = 257;
 static unsigned int defaultcs = 256;
-static unsigned int defaultrcs = 257;
+static unsigned int defaultrcs = 8;
 
 /*
  * Default shape of cursor
@@ -138,16 +137,15 @@
 /*
  * Default columns and rows numbers
  */
-
 static unsigned int cols = 80;
-static unsigned int rows = 24;
+static unsigned int rows = 30;
 
 /*
  * Default colour and shape of the mouse cursor
  */
 static unsigned int mouseshape = XC_xterm;
-static unsigned int mousefg = 7;
-static unsigned int mousebg = 0;
+static unsigned int mousefg = 257;
+static unsigned int mousebg = 256;
 
 /*
  * Color used to display font attributes when fontconfig selected a font which
