--- a/source/link/CMakeLists.txt	2020-09-24 15:53:49.000000000 +0100
+++ b/source/link/CMakeLists.txt	2020-10-06 10:45:32.457019359 +0100
@@ -11,7 +11,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-add_library(SPIRV-Tools-link STATIC
+add_library(SPIRV-Tools-link
   linker.cpp
 )
 
--- a/source/opt/CMakeLists.txt	2020-09-24 15:53:49.000000000 +0100
+++ b/source/opt/CMakeLists.txt	2020-10-06 10:45:54.161020836 +0100
@@ -233,7 +233,7 @@
 
 spvtools_pch(SPIRV_TOOLS_OPT_SOURCES pch_source_opt)
 
-add_library(SPIRV-Tools-opt STATIC ${SPIRV_TOOLS_OPT_SOURCES})
+add_library(SPIRV-Tools-opt ${SPIRV_TOOLS_OPT_SOURCES})
 
 spvtools_default_compile_options(SPIRV-Tools-opt)
 target_include_directories(SPIRV-Tools-opt
--- a/source/reduce/CMakeLists.txt	2020-09-24 15:53:49.000000000 +0100
+++ b/source/reduce/CMakeLists.txt	2020-10-06 10:46:19.718022575 +0100
@@ -78,7 +78,7 @@
 
 spvtools_pch(SPIRV_TOOLS_REDUCE_SOURCES pch_source_reduce)
 
-add_library(SPIRV-Tools-reduce STATIC ${SPIRV_TOOLS_REDUCE_SOURCES})
+add_library(SPIRV-Tools-reduce ${SPIRV_TOOLS_REDUCE_SOURCES})
 
 spvtools_default_compile_options(SPIRV-Tools-reduce)
 target_include_directories(SPIRV-Tools-reduce
