--- old/cmake/FindMsgpack.cmake	2023-03-04 21:11:01.542859024 +0000
+++ new/cmake/FindMsgpack.cmake	2023-03-04 21:11:10.837858989 +0000
@@ -7,7 +7,7 @@
 find_package(PkgConfig)
 if (PKG_CONFIG_FOUND)
   pkg_search_module(PC_MSGPACK QUIET
-    msgpackc>=${Msgpack_FIND_VERSION}
+    msgpack-c>=${Msgpack_FIND_VERSION}
     msgpack>=${Msgpack_FIND_VERSION})
 endif()
 
@@ -28,9 +28,9 @@
 
 if(MSVC)
   # The import library for the msgpack DLL has a different name
-  list(APPEND MSGPACK_NAMES msgpackc_import)
+  list(APPEND MSGPACK_NAMES msgpack-c_import)
 else()
-  list(APPEND MSGPACK_NAMES msgpackc msgpack)
+  list(APPEND MSGPACK_NAMES msgpack-c msgpack)
 endif()
 
 find_library(MSGPACK_LIBRARY NAMES ${MSGPACK_NAMES}
